### PR TITLE
feat: Add optional fastBurn/slowBurn aliases for page and ticket alerts

### LINKED ISSUE #833: Have optional new naming for `page` and `ticket` alert options
One of the most common painpoints we get from Sloth users is thinking the `ticket` alert option literally makes a ticket, like in Jira, while `page` will actually page them. When in [reality](https://github.com/slok/sloth/issues/489#issuecomment-1483564756) they distinguish the MWMB burn rate alerts.

We should make a corresponding, non-breaking new naming convention for these alert keys. One word options are nice so they are consistent in regular YAML and the Kubernetes spec. Some ideas:

* `fast` and `slow` (probably my favorite)
* `high` and `low`
* `fast_burn` and `long_burn`
* `rapid` and `sustained`

let me know any thoughts

### PR DESCRIPTION
**Pull request by cxdy**
Closes #833 

Accept fast_burn/slow_burn as non-breaking aliases of page_alert/ticket_alert and the CRD pageAlert/ticketAlert keys. Old names still work; setting both for one class is an error. Generated rules stay page/ticket.

### CODE REVIEW

**Comment by wbollock** [comment]
I prefer `fast_burn` and `slow_burn` compared to `high` and `low`. high and low makes it feel like they are related to severities or urgency rather than burn rate

File: pkg/kubernetes/api/sloth/v1/alerting.go [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+}
+
+func rejectExclusiveKeys(data []byte, canonical, alias string) error {
+	var raw map[string]json.RawMessage
+	if err := json.Unmarshal(data, &raw); err != nil {
+		return err
+	}
+	_, hasCanonical := raw[canonical]
+	aliasRaw, hasAlias := raw[alias]
+	if hasCanonical && hasAlias && !isEmptyAlertJSON(aliasRaw) {
```
Comment (wbollock) [inline pkg/kubernetes/api/sloth/v1/alerting.go]: AI-generated content prepared on Will's behalf.

The alias path fails an unstructured round-trip. `runtime.DefaultUnstructuredConverter.ToUnstructured` emits empty `pageAlert` and `ticketAlert` objects for zero-value struct fields. With only `FastBurn` set, the JSON contains both `pageAlert: {}` and `fastBurn: {...}`. This check then rejects the object at line 48.

I reproduced this with the PR source. Please add a round-trip test with `FastBurn` and `SlowBurn`, then treat an empty canonical object like an empty alias when checking exclusivity. The current test only covers a canonical `PageAlert` round-trip.

Blocking: yes.

File: pkg/kubernetes/api/sloth/v1/types.go [hunk 2]
```diff
@@ -156,11 +156,23 @@ type Alerting struct {
...
+	// PageAlert is the high-burn MWMB alert class (JSON/YAML key "pageAlert").
+	// fastBurn is an accepted alias of this key; pageAlert and fastBurn must not both be set.
 	PageAlert Alert `json:"pageAlert,omitempty"`
 
-	// TicketAlert alert refers to the warning alert (check multiwindow-multiburn alerts).
+	// TicketAlert is the low-burn MWMB alert class (JSON/YAML key "ticketAlert").
+	// slowBurn is an accepted alias of this key; ticketAlert and slowBurn must not both be set.
 	TicketAlert Alert `json:"ticketAlert,omitempty"`
+
+	// FastBurn is an optional alias of PageAlert (JSON/YAML key "fastBurn").
```
Comment (wbollock) [inline pkg/kubernetes/api/sloth/v1/types.go:167]: AI-generated content prepared on Will's behalf.

The new `FastBurn` and `SlowBurn` fields are missing from the generated apply configuration. `pkg/kubernetes/gen/applyconfiguration/sloth/v1/alerting.go` still has only `PageAlert` and `TicketAlert`, with no `WithFastBurn` or `WithSlowBurn` methods.

Users of the generated Kubernetes apply API cannot set the aliases, even though the typed API and CRD support them. Please regenerate the apply configuration and check the other generated files for the new fields.

Blocking: yes for users of the generated apply API.

File: pkg/kubernetes/api/sloth/v1/alerting.go (hunk 1 above)
Comment (cxdy) [reply pkg/kubernetes/api/sloth/v1/alerting.go]: Addressed in ca89348.

`rejectExclusiveKeys` now treats an empty canonical object the same as an empty alias, so unstructured round-trips that emit `pageAlert: {}` / `ticketAlert: {}` next to `fastBurn` / `slowBurn` succeed. Added `TestAlertingUnstructuredRoundTripKeepsFastBurn`.

File: pkg/kubernetes/api/sloth/v1/types.go (hunk 2 above)
Comment (cxdy) [reply pkg/kubernetes/api/sloth/v1/types.go:167]: Addressed in ca89348.

Regenerated the apply configuration with kube-code-generator v0.9.0. `AlertingApplyConfiguration` now has `FastBurn` / `SlowBurn` plus `WithFastBurn` / `WithSlowBurn`. Deepcopy, CRD, clientset, informers, and listers already included the new fields.

**Review by wbollock — APPROVED** [review]
lgtm!
